Xamarin.Forms.Platform.Android/Renderers/LabelRenderer.cs

```diff
@@ -169,6 +169,13 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateMaxLines();
 			else if (e.PropertyName == Label.PaddingProperty.PropertyName)
 				UpdatePadding();
+			else if (e.PropertyName == VisualElement.FlowDirectionProperty.PropertyName)
+				UpdateFlowDirection();
+		}
+
+		void UpdateFlowDirection()
```

Comment:
you need to call `UpdateFlowDirection()` on `OnElementChanged` too (line 125 and line 137)